string topicName
int64 nImages
---
sensor_msgs/Image[] images

